### PR TITLE
These were the relavent changes made to enable troposphere to log int…

### PR DESCRIPTION
**Pull request by rob-baron**
…o atmosphere

As of this commit, login, viewing the images and starting an image work.

Am checking this in now as we just setup the responsitories.

### CODE REVIEW

**Comment by xuhang57** [comment]
Thanks!

0, You might want to change the title of this PR. It could be shorter and put more details in the comments. 
1, We should not include the local.py which locates in the setting.py since it is merely settings for local. You could put put it in the git ingore.
2, We don't need the help page changes since I was taking this page for preparing a draft page for marketplace. 
3, We might want to refact the login a bit since we were hacking it and taking `external_link` as a learning example. Consequently, there are might some lines that we don't actually need, and some variables names needed to be changed so we won't confuse other people. 

This is the PR that Steve is working on: https://github.com/cyverse/troposphere/pull/524

And we can see what we can learn from it and what we should keep.

File: troposphere/settings/local.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+
+
+INSTALLED_APPS += (
+    'sslserver',
+)
+
+
+
+ALWAYS_AUTH_USER = "lucas"
```
Comment (kamfonik) [inline troposphere/settings/local.py:136]: Should this be here?

File: troposphere/static/js/components/help/HelpPage.jsx [hunk 2]
```diff
@@ -2,7 +2,62 @@ import React from "react";
 import stores from "stores";
 import globals from "globals";
 
+export default React.createClass({
+    displayName: "MarketplaceCorePage",
+    
+    render: function() {
+    
+    let imgs = ["https://info.massopencloud.org/wp-content/uploads/2016/03/bu-e1460748238801.jpg", "https://info.massopencloud.org/wp-content/uploads/2016/03/harvard.jpg", "https://info.massopencloud.org/wp-content/uploads/2016/03/mit.jpg", "https://info.massopencloud.org/wp-content/uploads/2016/03/northeastern.jpg"];
```
Comment (kamfonik) [inline troposphere/static/js/components/help/HelpPage.jsx:10]: If it's possible in this context, you might want some kind of BASE_IMG_URL='https://info.massopencloud.org/wp-content/uploads/' type thing for all these images.

File: troposphere/settings/local.py (hunk 1 above)
Comment (kamfonik) [inline troposphere/settings/local.py]: Should this be here?

**Review by kamfonik — COMMENTED** [review]
This looks good, left some comments.   +1 with the caveat that my knowledge of Javascript is pretty minimal.

File: troposphere/settings/local.py (hunk 1 above)
Comment (lucashxu) [reply troposphere/settings/local.py:136]: Shouldn't. we can safely leave this as blank now. It was a testing user and we moved forward. Thanks!

File: troposphere/static/js/components/help/HelpPage.jsx (hunk 2 above)
Comment (lucashxu) [reply troposphere/static/js/components/help/HelpPage.jsx:10]: good catch! I believe currently, we don' really need this page. I was using it as a placeholder for marketplace page.

**Review by xuhang57 — COMMENTED** [review]
I beleive, the LoginModal.react.js is putting under the wrong direction. It should be under /troposphere/static/js/.....

**Pull request closed without merging**